server/data/lockManager.ts

```diff
@@ -19,9 +19,15 @@ export default class LockManager {
     }
   }
 
-  public async lock(key: string, durationSeconds: number): Promise<boolean> {
+  public async lock(moveId: string, durationSeconds: number): Promise<boolean> {
     await this.ensureConnected()
-    const result = await this.client.set(`${this.prefix}${key}`, 'LOCKED', { NX: true, EX: durationSeconds })
+    const result = await this.client.set(`${this.prefix}${moveId}`, 'LOCKED', { NX: true, EX: durationSeconds })
+    return Boolean(result)
+  }
+
+  public async getLockStatus(moveId: string): Promise<boolean> {
```

Comment:
As discussed, wouldn't this be better named `isLocked`? 

getLockStatus and returning a boolean doesn't really make sense, it doesn't tell you what the boolean represents.

If you were to guess that the boolean represent whether a lock is present or not (arguably a big leap!), how would you know whether true represents locked or unlocked? - there is no context to help you know

Reply:
isLocked was used in the middleware function. I decided to have a diff name within the lock manager module because it makes searching for code a bit easier if things aren't named the same. 
But small matter so have changed to isLocked